### DISCUSSION TITLE
Why does isort consider `parser` as built-in package?

### DISCUSSION THREAD

**Discussion by secsilm** [PyCQA/isort#2290] [Q&A] [answered]
`parser` 总是与 `time` 和 `typing` 这些内置包在一起：

```python
import time
from parser import CategoryParser, TagParser  # <--- HERE
from typing import Tuple

from dateutil.parser import parse as parsedate
from loguru import logger

from data_types import CategoryResult, News, TagResult
from utils import clean_text
```

**Comment by guillermodotn** [comment, marked as answer]
Hello @secsilm,  

It depends on which Python version you are setting when using `isort`. If you're running Python **3.13**, you can try the following command:  

```bash
isort --py 313 <name_of_your_file.py>
```
This ensures that parser is correctly sorted for that Python version.

Why does this happen?

When you don't specify the Python version, `isort` creates a set containing all standard library packages available across all **Python 3.X** versions. Since parser was removed in **Python 3.10**, it is still included when `isort` considers older versions.

**Reply by secsilm** [reply]
ok, thanks.
